### PR TITLE
Addresses issue 267 by adding yaml template for operator v2.X

### PR DESCRIPTION
**Pull request by dobhaweim**
Signed-off-by: David O'Connor <davidon5@ie.ibm.com>

### CODE REVIEW

**Comment by prudhvigodithi** [comment]
Hey @dobhaweim thanks for the PR can we add the latest version `2.3.0`?

**Comment by dobhaweim** [comment]
@prudhvigodithi sure thing, done.

File: opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  security:
+    config:
+    tls:
+       http:
+         generate: true 
+       transport:
+         generate: true
+         perNode: true
+  general:
+    httpPort: 9400
```
Comment (swoehrl-mw) [inline opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml:18]: Any reason this is not the default `9200`?

File: opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+         cpu: "200m"
+      limits:
+         memory: "512Mi"
+         cpu: "200m"
+  nodePools:
+    - component: masters
+      replicas: 3
+      resources:
+         requests:
+            memory: "4Gi"
```
Comment (swoehrl-mw) [inline opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml:42]: Resources are quite high for a minimal configuration (the other example has 2Gi and 500m)

**Review by swoehrl-mw — CHANGES_REQUESTED** [review]

File: opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml (hunk 2 above)
Comment (dobharweim) [reply opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml:42]: I got a pod eviction due to 137 memory error on IKS with the other example, numerous times.

File: opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml (hunk 1 above)
Comment (dobharweim) [reply opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml:18]: Not that I recall! I will revert.

File: opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml (hunk 2 above)
Comment (dobhaweim) [reply opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml:42]: I tried numerous times with the lower resources on IKS but the pod was repeated killed by the scheduler for exceeding memory limits (137 status).

**Comment by swoehrl-mw** [comment]
@dobhaweim Can you update your commits to comply with the DCO, then the PR is fine for me.

**Comment by dobhaweim** [comment]
> @dobhaweim Can you update your commits to comply with the DCO, then the PR is fine for me.

@swoehrl-mw sure! I made a commit through the Github UI which is signed with a GPG key if you check the commit itself. However I see it is failing the DCO check. I reverted that commit and recommitted with an appropriate signoff. But DCO is still showing red for me? Does this just need to be rerun?

**Comment by swoehrl-mw** [comment]
@dobhaweim The revert still leaves the old commit (0dfadaec6f6ee936c874acf9420f76927a2470ce) in the history. You need to edit the old commit and then force-push to your branch.

**Comment by dobhaweim** [comment]
I'm following the Git docs but seem to be adding _more_ unsigned commits 😆 

Furthermore - I sent the original commit here from an old account accidentally while migrating from an old student email account about to be shut down.

As such I have made the ignoble decision to kill this PR and do a new one with my regular GH account and we can all pretend the thing with the sign offs did not happen.

**Pull request closed without merging**